src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -37,3 +37,11 @@
 .proteinFeaturesCountWrapper {
   display: inline-block;
 }
+
+.loadingContainer {
+  height: 100%;
```

Comment:
Given this:

![image](https://user-images.githubusercontent.com/6834224/95105515-2b862100-072f-11eb-9965-fd8605634f15.png)

what's the purpose of height: 100% and flex-direction: column?

Reply:
Removed them